frontend/src/components/RideModal/Pages/RideTimes.tsx

```diff
@@ -47,6 +52,12 @@ const RideTimesPage = ({ formData, onSubmit }: ModalPageProps) => {
               },
             })}
           />
+          {errors.dropoffTime?.type === 'required' && (
+            <p className={styles.error}>Please enter a time</p>
+          )}
+          {errors.dropoffTime?.type === 'validate' && (
+            <p className={styles.error}>Invalid time</p>
```

Comment:
Should we also add another error that indicates if the entered time is in the past (so only allow scheduling a ride in the future)?

Reply:
yeah we'd probably want that, good find